### PR TITLE
Add fields to `NodeStatus` and `Event`

### PR DESCRIPTION
**Pull request by kimhanbeom**
- Added the fields of the `Event` enum internal struct provided via GraphQL. The added fields are used for more detailed `detect event filtering`.
- Added a `ping` field to `NodeStatus` struct. The added fields are available through the `NodeStatusList` query.
- Modify `status::laod` function so that the `NodeStatusList` query returns a ping field.

reference: https://github.com/aicelab/sasui/issues/2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,20 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
...
+### Added
+
+- Added the fields of the `Event` enum internal struct provided via GraphQL.
+  The added fields are used for more detailed `detect event filtering`.
+- Added a `ping` field to `NodeStatus` struct. The added fields are available
+  through the `NodeStatusList` query.
+
+### Changed
+
+- Modify `status::laod` function so that the `NodeStatusList` query returns
```
Comment (msk) [inline CHANGELOG.md:19]: `laod` -> `load`

Comment (kimhanbeom) [reply CHANGELOG.md]: 오타 수정 완료했습니다

**Review by syncpark — APPROVED** [review]

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "review-web"
-version = "0.10.0"
+version = "0.11.0-alpha.1"
```
Comment (msk) [inline Cargo.toml:3]: 하위호환성을 깨는 변경 내용이 있나요? 보통 GraphQL API에 뭔가 추가한 건 하위호환이 되는데요.

Comment (kimhanbeom) [reply Cargo.toml]: 해당 사항 수정 하였습니다

**Comment by msk** [comment]
Rebased to resolve conflicts with main.

**Pull request merged**